### PR TITLE
add daily star support

### PR DESCRIPTION
**Pull request by Jabbawukis**

### CODE REVIEW

File: src/fundus/publishers/uk/daily_star.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.meta.get("author"))
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def topics(self) -> List[str]:
+            return generic_topic_parsing(self.precomputed.meta.get("article:tag"))
```
Comment (MaxDall) [inline src/fundus/publishers/uk/daily_star.py:42]:
```suggestion
            return generic_topic_parsing(self.precomputed.meta.get("keywords"))
```

File: src/fundus/publishers/uk/daily_star.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class DailyStarParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("p.sub-title")
+        _paragraph_selector = CSSSelector("div.article-body")
```
Comment (MaxDall) [inline src/fundus/publishers/uk/daily_star.py:18]:
```suggestion
        _paragraph_selector = XPath("//div[@class='article-body'] /p[text()]")
```

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding :)

**Comment by Jabbawukis** [comment]
fix Xpath import and update unit tests -> checks pass now

**Review by MaxDall — APPROVED** [review]

**Pull request merged**